Makefile

```diff
@@ -361,6 +361,10 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
 FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
 endif
 
+.PHONY: catalog-update
+catalog-update:
```

Comment:
pls add `## Generate catalog yaml for konflux` or something like that to show in 
`make help`